tests/math/math_utils_test.py

```diff
@@ -0,0 +1,27 @@
+# Copyright OTT-JAX
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import jax
+import jax.numpy as jnp
+from ott.math import utils as mu
+
+
+class TestNorm:
+
+  def test_norm(self, rng: jax.random.PRNGKeyArray):
+    d = 13
+    f = lambda x: mu.norm(x - x)
+    g = lambda x: jnp.linalg.norm(x - x)
+    x = jax.random.uniform(rng, (d,))
+    assert jnp.all(jax.grad(f)(x) == 0.0)
```

Comment:
Please also add a test for `jax.hessian`. [UPDATE] Running:
```python
import jax
import jax.numpy as jnp
from ott.math import utils as mu

rng = jax.random.PRNGKey(0)

d = 100
f = lambda x: mu.norm(x)
g = lambda x: jnp.linalg.norm(x)
x = jax.random.uniform(rng, (d,))

h_g = jax.hessian(g)(x)
h_f = jax.hessian(f)(x)

jnp.max(jnp.abs(h_f - h_g))  # Array(0.16832732, dtype=float32)
```
doesn't seem to hold. I guess this is to be expected, if we define the custom JVP on the `norm`; maybe having a custom JVP for `def h(z: jnp.ndarray): ...` would solve this?

Reply:
probably due to my bug before! now passes